src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md

```diff
@@ -3,8 +3,21 @@ title: Import a Google Font
 ---
 ## Import a Google Font
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+````CSS
+ <link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
+
+<style>
+  .red-text {
+    color: red;
+  }
+  h2 {
+    font-family: Lobster;
+  }
+  p {
+    font-size: 16px;
+    font-family: monospace;
+  }
+</style>
+
```

Comment:
Add the missing syntax to close the codeblock.

Reply:
Can you please check if i have done it right.